spec/system/teacher_interface/application_spec.rb

```diff
@@ -685,266 +710,483 @@ def and_i_receive_an_application_email
       "This is the qualification that led to you being recognised as a teacher."
     )
   end
 
   def then_i_see_the_degree_qualifications_form
-    expect(page).to have_title("Your qualifications")
-    expect(page).to have_content("Your university degree")
-    expect(page).to have_content(
+    expect(qualifications_form_page).to have_title("Your qualifications")
+    expect(qualifications_form_page.heading.text).to have_content(
+      "Your university degree"
+    )
+    expect(qualifications_form_page.body).to have_content(
       "Tell us about your university degree qualification."
     )
   end
 
   def then_i_see_the_upload_certificate_form
-    expect(page).to have_title("Upload a document")
-    expect(page).to have_content("Upload your certificate for Name")
+    expect(upload_document_page).to have_title("Upload a document")
+    expect(upload_document_page.heading.text).to have_content(
+      "Upload your certificate for Name"
+    )
   end
 
   def then_i_see_the_upload_degree_certificate_form
-    expect(page).to have_title("Upload a document")
-    expect(page).to have_content("Upload your certificate for Name")
+    expect(upload_document_page).to have_title("Upload a document")
+    expect(upload_document_page.heading.text).to have_content(
+      "Upload your certificate for Name"
+    )
   end
 
   def then_i_see_the_upload_transcript_form
-    expect(page).to have_title("Upload a document")
-    expect(page).to have_content("Upload your transcript for Name")
+    expect(upload_document_page).to have_title("Upload a document")
+    expect(upload_document_page.heading.text).to have_content(
+      "Upload your transcript for Name"
+    )
   end
 
   def then_i_see_the_upload_degree_transcript_form
-    expect(page).to have_title("Upload a document")
-    expect(page).to have_content("Upload your transcript for Name")
+    expect(upload_document_page).to have_title("Upload a document")
+    expect(upload_document_page.heading.text).to have_content(
+      "Upload your transcript for Name"
+    )
   end
 
   def then_i_see_the_age_range_form
-    expect(page).to have_title("Enter the age range you can teach")
-    expect(page).to have_content("What age range are you qualified to teach?")
-    expect(page).to have_content("From")
-    expect(page).to have_content("To")
+    expect(age_range_form).to have_title("Enter the age range you can teach")
+    expect(age_range_form.heading.text).to have_content(
+      "What age range are you qualified to teach?"
+    )
+    expect(age_range_form.grid).to have_content("From")
+    expect(age_range_form.grid).to have_content("To")
   end
 
   def then_i_see_the_subjects_form
-    expect(page).to have_title("Enter the subjects you can teach")
-    expect(page).to have_content("What subjects are you qualified to teach?")
+    expect(subjects_form_page).to have_title("Enter the subjects you can teach")
+    expect(subjects_form_page.heading.text).to have_content(
```

Comment:
```suggestion
    expect(subjects_form_page.heading).to have_content(
```

If you're using `have_content` you don't need to do `.text` as we're checking the content of the element. If you would like to check the text directly you should use `eq`.